vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
Yes that's my point, we *don't* want different versions. But if we put Nick's code in the dummy app, we might get different versions because react is included from a different gem.